pa4/checker-pa4.ml

```diff
@@ -481,12 +512,13 @@ let rec exp_typecheck(o_e: obj_env) (m_e: method_env) (exp: expression) : static
 					method_identifier_line_number method_name caller_type_name ;
 			exit 1
 		end
-	end
-	| New(class_identifier) -> begin
-		printf "Doing a New\n" ;
-		let (_, class_name) = class_identifier in
-		Class(class_name)
-	end
+	end	
+	| New(e1) -> 
```

Comment:
Rather than calling this `e1`, which would imply that this was a expression, I think that we should call this `class_identifier` since that would be clearer.